azure-pipelines.yml

```diff
@@ -171,56 +171,3 @@ stages:
         pool:
           name: NetCoreInternal-Pool
           queue: buildpool.windows.10.amd64.vs2017
-
-  - template: /eng/stages/publish.yml
```

Comment:
We need the template to trigger Arcade publish, I think the smallest change is to give `dependsOnPublishStages: []` here.

Reply:
It defaults to `[]`, just omit params and should be good.